vm/vmware/vmware.go

```diff
@@ -119,11 +119,21 @@ func (inst *instance) boot() error {
 	if inst.debug {
 		log.Logf(0, "getting IP of %v", inst.vmx)
 	}
-	ip, err := osutil.RunCmd(5*time.Minute, "", "vmrun", "getGuestIPAddress", inst.vmx, "-wait")
+	ipCmd := exec.Command("vmrun", "getGuestIPAddress", inst.vmx, "-wait")
+	ipOutput, err := ipCmd.CombinedOutput()
```

Comment:
Does the tool print the warning to stderr?

If yes, it might be easier to take the stdout output, i.e. `ip, err := ipCmd.Output()` instead of `ipCmd.CombinedOutput()`.